FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Views/PasswordPromptView.swift

```diff
@@ -9,26 +10,47 @@ struct PasswordPromptSheet {
 extension PasswordPromptSheet: View {
   var body: some View {
     VStack(spacing: 20) {
-      SecureField(authService.string.passwordInputLabel, text: $password)
-        .textFieldStyle(.roundedBorder)
+      Text(authService.string.confirmPasswordInputLabel)
```

Comment:
Improved the UI for PasswordPromptSheet:
<img src="https://github.com/user-attachments/assets/1e417264-d858-49d5-91b3-4913cc2e87f1" width="200px" />